streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignorTest.java

```diff
@@ -483,7 +485,7 @@ public void testEagerSubscription() {
         Collections.sort(subscription.topics());
         assertEquals(asList("topic1", "topic2"), subscription.topics());
 
-        final SubscriptionInfo info = getInfo(UUID_1, prevTasks, standbyTasks);
```

Comment:
Can you add a test that verifies that it goes back and forth between the two expected values when you call `partitionAssignor.subscriptionUserData` multiples times (let's say 3) -- also maybe add a verification that the `uniqueField` has a length of just 1

Reply:
> goes back and forth between the two expected values 
What do you mean by expected values? Toggling between the default of `0` and `1` after `partitionAssignor.subscriptionUserData` is called the first time?

Reply:
Yeah, it should be `0` the first time you call it, then `1` the second time, and then back to `0` again on the third call

Reply:
I don't think it would go from `0` to `1` to `0` since the partition assigner always increases, so if you're using the same partition assignor it'll go from `0` to `1` to `2` etc

Reply:
Oh, saw that you already added the test for `0` --> `1` --> `2` behavior. In that case can you just add two things to that test: verify the behavior on overflow, and verify that the length is always exactly one byte